docs/user_guide.md

```diff
@@ -489,8 +489,21 @@ checker configuration:
                         WISELY AND AT YOUR OWN RISK!
 ~~~~~~~~~~~~~~~~~~~~~
 
-Both `--enable` and `--disable` take individual checkers or checker groups as
-their argument and there can be any number of such flags specified.
+Both `--enable` and `--disable` take individual checkers, checker groups or
```

Comment:
Maybe a side note for the users would be useful that the order of the checkers/groups/profiles at the command line is important.
If a checker is enabled but the group or profile is disabled later (to which the checker belongs to) it will be disabled at the analysis.
I think we wrote about this somewhere but I think it would be nice to highlight it here too.